pf/tests/provider_check_test.go

```diff
@@ -133,7 +133,7 @@ func TestCheck(t *testing.T) {
 			  },
 			  "response": {
                             "inputs": {},
-                            "failures": [{"property": "prop", "reason": "Missing a required property"}]
```

Comment:
Not using "property" seems to bring PF in line with how SDKv2 providers behaved. This has some implications on UX/ display through the CLI. We can reconsider later as a separate change.